packages/x-components/src/design-system/components/suggestion/default.tokens.scss

```
@@ -8,6 +8,12 @@
   --x-color-text-suggestion-default-matching: var(--x-color-text-suggestion-default);
   --x-color-background-suggestion-default: transparent;
   --x-color-border-suggestion-default: var(--x-color-text-suggestion-default);
+  // color curated
+  --x-color-text-suggestion-default-curated: var(--x-color-text-default);
+  --x-color-text-suggestion-matching-part-default-curated: var(--x-color-base-neutral-35);
+  --x-color-text-suggestion-default-matching-curated: var(--x-color-text-suggestion-default);
+  --x-color-background-suggestion-default-curated: transparent;
+  --x-color-border-suggestion-default-curated: var(--x-color-text-suggestion-default);
```

These tokens should be referencing their default counterparts:

```suggestion
  --x-color-text-suggestion-default-curated: var(--x-color-text-suggestion-default);
  --x-color-text-suggestion-matching-part-default-curated: var(
    --x-color-text-suggestion-matching-part-default
  );
  --x-color-text-suggestion-default-matching-curated: var(
    --x-color-text-suggestion-default-matching
  );
  --x-color-background-suggestion-default-curated: var(--x-color-background-suggestion-default);
  --x-color-border-suggestion-default-curated: var(--x-color-border-suggestion-default);
```